packages/web/src/app/shared/doc-pages.ts

```diff
@@ -64,6 +64,16 @@ export const componentsDocPages: DocPage[] = [
     searchTerms: ['notifications', 'feedback', 'banner', 'flag', 'snackbar', 'message', 'confirmation'],
     relatedPages: ['toast', 'badge', 'cookies', 'validations'],
   },
+  {
+    title: 'App Bridge',
+    description:
+      'The app bridge can be used to open the selected meteringpoint in a different Elvia application. The list of application-links can be modified based on the systems needs. ',
```

Comment:
How can the links be modified? Am I missing something?

Reply:
True, it can't. Purely copy-paste from Figma 🥴